ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -94,6 +94,11 @@ instances:
     # channels:
     #   - <CHANNEL_NAME>
 
+    ## @param auto_discover_channels - boolean - optional - default: true
+    ## Autodiscover channels to monitor. This will discover all discoverable channels.
```

Comment:
To reviewer: is there a more thorough description I can use instead of this?

Reply:
The description is good, it's nice and concise.

Reply:
```suggestion
    ## Autodiscover channels to monitor. This finds all discoverable channels.
```